src/static/css/2019.css

```diff
@@ -1223,3 +1223,97 @@ p.copyright a {
     margin-top: 20px;
   }
 }
+
+.nav-dropdown {
+  position: relative;
+  margin-right: 2vw;
+}
+
+.nav-dropdown-btn {
+  padding: 20px 45px 20px 20px;
+  font-size: 17px;
+  font-size: 1.0625rem;
+  text-decoration: none;
+  position: relative;
+  display: flex;
+  align-items: center;
+  border: none;
+  border-radius: 25px;
+  background: #44536a;
+  color: white;
+}
+
+.nav-dropdown-btn.dropdown-open {
+  border-radius: 25px 25px 0 0;
+}
+
+.nav-dropdown-btn::after {
+  content: "";
+  position: absolute;
+  pointer-events: none;
+  border-style: solid;
+  border-width: 2px 2px 0 0;
+  border-color: currentColor;
+  height: 7px;
+  width: 7px;
+  top: 50%;
+  margin-top: -7px;
+  right: 23px;
+  -webkit-transform: rotate(135deg);
+  -ms-transform: rotate(135deg);
+  transform: rotate(135deg);
+  transition: all 200ms linear;
```

Comment:
Love this! 😍

Reply:
Think 100ms is enough? Seems a little laggy with 200ms.

Reply:
Could be fine tuned.